test/unit/lib/serverless.test.js

```diff
@@ -174,4 +177,181 @@ describe('test/unit/lib/serverless.test.js', () => {
       expect(serverless.config).to.have.property('servicePath');
     });
   });
+
+  describe('Extend configuration', () => {
+    let serverless;
+    let parseEntriesStub;
+
+    before(() => {
+      parseEntriesStub = sinon.stub().returns(new Map([]));
+      const ServerlessProxy = proxyquire('../../../lib/serverless.js', {
+        './configuration/variables/resolve-meta': {
+          parseEntries: parseEntriesStub,
+        },
+      });
+
+      serverless = new ServerlessProxy({
```

Comment:
Let's rely on `runServerless` util as above. We can pass there mocked modules as documented here https://github.com/serverless/test/blob/main/docs/run-serverless.md#modulescachestub (see also existing tests that do that)

Reply:
Done.